src/test/resources/org/broadinstitute/hellbender/tools/copynumber/gcnv-postprocess/intervals_output_SAMPLE_000.vcf

```diff
@@ -5,667 +5,667 @@
-X	2856133	CNV_X_2856133_2856300	N	<DEL>,<DUP>	.	.	END=2856300	GT:CN:CNLP:CNQ	0:1:312,0,187,246,288,312:187
-X	2861103	CNV_X_2861103_2861242	N	<DEL>,<DUP>	.	.	END=2861242	GT:CN:CNLP:CNQ	0:1:312,0,187,242,277,304:187
-X	2864036	CNV_X_2864036_2864177	N	<DEL>,<DUP>	.	.	END=2864177	GT:CN:CNLP:CNQ	0:1:312,0,172,252,292,312:172
-X	2867342	CNV_X_2867342_2867770	N	<DEL>,<DUP>	.	.	END=2867770	GT:CN:CNLP:CNQ	0:1:312,0,178,254,286,310:178
-X	2871181	CNV_X_2871181_2871308	N	<DEL>,<DUP>	.	.	END=2871308	GT:CN:CNLP:CNQ	0:1:312,0,168,215,244,268:168
-X	2873454	CNV_X_2873454_2873580	N	<DEL>,<DUP>	.	.	END=2873580	GT:CN:CNLP:CNQ	0:1:312,0,177,203,223,240:177
-X	2876312	CNV_X_2876312_2876478	N	<DEL>,<DUP>	.	.	END=2876478	GT:CN:CNLP:CNQ	0:1:312,0,169,237,272,301:169
-X	2878416	CNV_X_2878416_2878443	N	<DEL>,<DUP>	.	.	END=2878443	GT:CN:CNLP:CNQ	0:1:312,0,176,201,220,238:176
-X	2924651	CNV_X_2924651_2924747	N	<DEL>,<DUP>	.	.	END=2924747	GT:CN:CNLP:CNQ	0:1:312,0,165,241,280,309:165
-X	2928068	CNV_X_2928068_2928194	N	<DEL>,<DUP>	.	.	END=2928194	GT:CN:CNLP:CNQ	0:1:312,0,192,252,286,310:192
-X	2931085	CNV_X_2931085_2931215	N	<DEL>,<DUP>	.	.	END=2931215	GT:CN:CNLP:CNQ	0:1:312,0,199,218,237,254:199
-X	2933008	CNV_X_2933008_2933436	N	<DEL>,<DUP>	.	.	END=2933436	GT:CN:CNLP:CNQ	0:1:312,0,201,255,293,312:201
-X	2936572	CNV_X_2936572_2936713	N	<DEL>,<DUP>	.	.	END=2936713	GT:CN:CNLP:CNQ	0:1:312,0,178,255,290,312:178
-X	2942059	CNV_X_2942059_2942198	N	<DEL>,<DUP>	.	.	END=2942198	GT:CN:CNLP:CNQ	0:1:312,0,175,242,270,295:175
-X	2945351	CNV_X_2945351_2945518	N	<DEL>,<DUP>	.	.	END=2945518	GT:CN:CNLP:CNQ	0:1:312,0,191,258,303,312:191
-X	2947285	CNV_X_2947285_2947411	N	<DEL>,<DUP>	.	.	END=2947411	GT:CN:CNLP:CNQ	0:1:312,0,167,235,279,309:167
-X	2951056	CNV_X_2951056_2951428	N	<DEL>,<DUP>	.	.	END=2951428	GT:CN:CNLP:CNQ	2:3:312,225,82,0,44,105:44
-X	2986139	CNV_X_2986139_2986154	N	<DEL>,<DUP>	.	.	END=2986154	GT:CN:CNLP:CNQ	0:1:312,0,182,235,281,312:182
-X	2990064	CNV_X_2990064_2990218	N	<DEL>,<DUP>	.	.	END=2990218	GT:CN:CNLP:CNQ	0:1:312,0,199,222,242,260:199
-X	2994586	CNV_X_2994586_2994712	N	<DEL>,<DUP>	.	.	END=2994712	GT:CN:CNLP:CNQ	0:1:312,0,196,272,308,312:196
-X	2998929	CNV_X_2998929_2999056	N	<DEL>,<DUP>	.	.	END=2999056	GT:CN:CNLP:CNQ	0:1:312,0,170,238,270,299:170
-X	3002281	CNV_X_3002281_3002709	N	<DEL>,<DUP>	.	.	END=3002709	GT:CN:CNLP:CNQ	0:1:312,0,159,233,270,301:159
-X	3007534	CNV_X_3007534_3007675	N	<DEL>,<DUP>	.	.	END=3007675	GT:CN:CNLP:CNQ	0:1:312,0,195,247,278,303:195
-X	3019125	CNV_X_3019125_3019264	N	<DEL>,<DUP>	.	.	END=3019264	GT:CN:CNLP:CNQ	0:1:312,0,166,233,264,290:166
-X	3021800	CNV_X_3021800_3021967	N	<DEL>,<DUP>	.	.	END=3021967	GT:CN:CNLP:CNQ	0:1:312,0,171,247,285,312:171
-X	3028166	CNV_X_3028166_3028295	N	<DEL>,<DUP>	.	.	END=3028295	GT:CN:CNLP:CNQ	0:1:312,0,200,266,301,312:200
-X	3030212	CNV_X_3030212_3030599	N	<DEL>,<DUP>	.	.	END=3030599	GT:CN:CNLP:CNQ	0:1:312,0,188,257,29
```

Comment:
These changes seem non-trivial...why are the results different?  And why do the new results look so weird (lots of 100 CNLPs and CNQs)?

Reply:
I am not sure what count files were used to make previous test data - I could be using different ones. The field values are very strange, I'm looking into that